src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -131,10 +131,12 @@ public function __construct(
     /**
      * @Given there is a customer :customer that placed an order :orderNumber
      * @Given there is another customer :customer that placed an order :orderNumber
+     * @Given there is a customer :customer that placed an order
+     * @Given there is another customer :customer that placed an order
```

Comment:
We can write this in Turnip:

`@Given there is a|another customer :customer that placed an order`